src/task.c

```diff
@@ -1168,6 +1168,40 @@ JL_DLLEXPORT jl_task_t *jl_get_current_task(void)
     return pgcstack == NULL ? NULL : container_of(pgcstack, jl_task_t, gcstack);
 }
 
+extern int gc_first_tid;
+// Builds a list of the live tasks. Racy: `live_tasks` can expand at any time.
+arraylist_t *jl_get_all_tasks_arraylist(void) JL_NOTSAFEPOINT
```

Comment:
Should this go into the gc-stacks.c file, since it is basically the same as `jl_live_tasks` there?

Reply:
Done.